src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Microsoft.AspNetCore.Razor.Language.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <ProjectReference Include="$(SharedSourceRoot)\Microsoft.AspNetCore.Razor.LanguageSupport\Microsoft.AspNetCore.Razor.LanguageSupport.csproj" />
+    <PackageReference Include="System.Collections.Immutable" />
```

Comment:
Should this have a version?

Reply:
https://github.com/dotnet/razor/pull/8234/files#diff-25f8191744ef1ba7ffcb791f235b061657a04ca3f1a7af451ea2f5cd566e26b0R32

🙂

Reply:
I'd forgotten about that. Thanks!